src/objective-c/tests/Podfile

```diff
@@ -34,3 +34,18 @@ end
 target 'InteropTestsLocalCleartext' do
 	shared_pods
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['GCC_TREAT_WARNINGS_AS_ERRORS'] = 'YES'
+    end
+    if target.name == 'gRPC'
+      target.build_configurations.each do |config|
+        # GPR_UNREACHABLE_CODE causes "Control may reach end of non-void
```

Comment:
Let's open an issue to:
* fix `GPR_UNREACHABLE_CODE` so warnings don't show up, and 
* remove this setting.